src/stratis_cli/_error_reporting.py

```diff
@@ -153,7 +169,7 @@ def _interpret_errors(errors):
             # pylint: disable=bad-continuation
             isinstance(error, dbus.exceptions.DBusException)
             and error.get_dbus_name() == "org.freedesktop.DBus.Error.AccessDenied"
-        ):
+        ):  # pragma: no cover
```

Comment:
This should get  a FIXME, too. We can get this easily by running without adequate permissions.